packages/paste-core/components/sidebar/src/navigation/SidebarNavigation.tsx

```diff
@@ -47,13 +48,25 @@ const SidebarNavigationWrapper = styled.div(({theme}: {theme: ThemeShape}) => {
 
 export interface SidebarNavigationProps extends React.HTMLAttributes<HTMLDivElement> {
   children: React.ReactNode;
+  'aria-label': string;
   element?: BoxProps['element'];
   hideItemsOnCollapse?: boolean;
   hierarchical?: boolean;
+  sidebarNavigationSkipLinkID: string;
 }
 
 export const SidebarNavigation = React.forwardRef<HTMLDivElement, SidebarNavigationProps>(
-  ({element = 'SIDEBAR_NAVIGATION', hideItemsOnCollapse = false, hierarchical = false, children, ...props}, ref) => {
+  (
+    {
+      element = 'SIDEBAR_NAVIGATION',
+      hideItemsOnCollapse = false,
+      hierarchical = false,
+      sidebarNavigationSkipLinkID,
```

Comment:
thought: theoretically since we have SidebarContext, we don't need to pass an `sidebarNavigationSkipLinkID` at all since we can manage it ourselves from within Sidebar. Do you think it's valuable to make users have to pass this one?

Reply:
Oh, that's actually a great idea